apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/FissClaimRdaSinkTest.java

```diff
@@ -2,8 +2,7 @@
 
 import static gov.cms.bfd.pipeline.rda.grpc.RdaPipelineTestUtils.*;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
It's good practice to avoid star imports (and adjust your IDE if this was automatically done; you can set the "number of imports before using * imports" to 99 in intelliJ, and probably in other IDEs)

Reply:
Honestly I disagree but if that is the team's policy I'll make the change.

Reply:
Interesting that google format doesn't care about the star imports.